### ISSUE TITLE
Well done uplink and downlink budget calculations

### ISSUE THREAD

**Issue by knightss27** [UChicago-PULSE/mission-documentation#11]
Main textbook for reference: https://drive.google.com/drive/u/0/folders/1GSn6hGQgfGP_xs0WhW5sX6Fum4cg-Ott (HEMMATI)

**Comment by aadarwal** [comment]
Ground station team calculations from last week:

![Image](https://github.com/UChicago-PULSE/documentation/assets/83266031/5c1ec889-8e47-4eb0-a072-bc3d190a8236)



![Image](https://github.com/UChicago-PULSE/documentation/assets/83266031/d726619a-2d99-41a9-960e-654ffb0f2b17)

**Comment by knightss27** [comment]
Do we have any clue what these actually showed us...

**Comment by aadarwal** [comment]
> Do we have any clue what these actually showed us...

Yes yes I will add that, just wanted to put the pics up here so that it's not lost in chats in the discord.
